composer.json

```diff
@@ -84,26 +79,46 @@
                     "reference": "3.0.3"
                 }
             }
+        },
+        "deskpro/deskpro-api-client-php": {
+          "type": "package",
+          "package": {
+            "name": "deskpro/deskpro-api-client-php",
+            "version": "4.1.0",
+            "type": "library",
+            "source": {
+              "url": "https://github.com/itk-dev/deskpro-api-client-php.git",
+              "type": "git",
+              "reference": "hotfix/change-psr-log-dependency"
+            },
+            "autoload": {
+              "psr-0": {
+                "Deskpro\\": "src/"
+              }
+            }
+          }
         }
     },
     "require": {
         "php": "^8.1.0",
         "bower-asset/jquery-simple-color": "^1.2",
-        "bower-asset/jqueryui-touch-punch": "dev-master",
+        "bower-asset/jqueryui-touch-punch": "dev-gh-pages",
```

Comment:
This seems weird. The branch `gh-pages` is usually used for https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site#next-steps. But we have to assume/hope that they know what they're doing.